File: csharp/rna-transcription/RnaTranscription.cs
using System;

public static class RnaTranscription
{
    public static string ToRna(string nucleotide)
    {
        char []dna = nucleotide.ToCharArray();
        char []rna = new char[nucleotide.Length];
        for (int i = 0; i < nucleotide.Length; i++)
        {
            switch (dna[i])
            {
                case 'G':
                    rna[i] = 'C';
                    break;

                case 'C':
                    rna[i] = 'G';
                    break;

                case 'T':
                    rna[i] = 'A';
                    break;

                case 'A':
                    rna[i] = 'U';
                    break;
            }
        }
        return new string(rna);
    }
}

File: javascript/rna-transcription/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

const transcript = {
    'G': 'C',
    'C': 'G',
    'T': 'A',
    'A': 'U'
}

export const toRna = (dnaChain) => {
    return dnaChain.split('').map(dna => transcript[dna]).join('');
};
